src/runtime/virtcontainers/utils/utils_darwin.go

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2022 Apple Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package utils
+
+func GetDevicePathAndFsTypeOptions(mountPoint string) (devicePath, fsType string, fsOptions []string, err error) {
+	return
```

Comment:
I get that this is a stub to be expanded later, but this doesn't even build, does it? Wouldn't it be better to return empty, but language-correct values?

Reply:
It builds -- it'll have default empty values, since the return values are named in the function signature.

Reply:
ah, thanks -- I tested this on my side with a `bool`, but it does work with those types